Algorithm.Framework/Selection/UniverseSelectionModelPythonWrapper.cs

```diff
@@ -17,6 +17,7 @@
 using QuantConnect.Data.UniverseSelection;
 using System;
 using System.Collections.Generic;
+using QuantConnect.Interfaces;
```

Comment:
Is this new include required somehow?

Reply:
Yes, it's needed to resolve `IAlgorithm.Initialize` on this line:
https://github.com/QuantConnect/Lean/blob/57d50c69c43f46153e53d98c98b37f80b419048e/Algorithm.Framework/Selection/UniverseSelectionModelPythonWrapper.cs#L70